src/connectedk8s/HISTORY.rst

```diff
@@ -2,6 +2,12 @@
 
 Release History
 ===============
+1.3.15
+++++++
+
+* Diagnoser Enhancements - storing metadata and KAP CR snapshots , azure-arc helm values , azure-arc ns secret list
+* Removing circular imports of 1. custom from precheckutils and 2.(precheckutils and troubleshootutils) from utils
```

Comment:
Update about the force delete update in connect command also

Reply:
Done.